website/content/en/pre-docs/concepts/_index.md

```diff
@@ -0,0 +1,167 @@
+
+## Cluster operator
+
+As a Kubernetes cluster operator, you can engage with Karpenter to:
+
+* Install Karpenter 
+* Configure provisioners to set constraints and other features for managing nodes
+* Deprovision nodes
+* Upgrade nodes
+
+Concepts associated with this role are described below.
+
+
+### Installing Karpenter
+
+Karpenter is designed to run on a node in your Kubernetes cluster.
+As part of the installation process, you need credentials from the underlying cloud provider to allow nodes to be started up and added to the cluster as they are needed.
+
+[Getting Started with Karpenter on AWS](https://karpenter.sh/docs/getting-started/)
+describes the process of installing Karpenter on an AWS cloud provider.
+Because requests to add and delete nodes and schedule pods are made through Kubernetes, AWS IAM Roles for Service Accounts (IRSA) are needed by your Kubernetes cluster to make privileged requests to AWS.
+For example, Karpenter uses AWS IRSA roles to grant the permissions needed to describe EC2 instance types and create EC2 instances.
+
+Once privileges are in place, Karpenter is deployed with a Helm chart.
+
+### Configuring provisioners
+
+Karpenter's job is to add nodes to handle unschedulable pods, schedule pods on those nodes, and remove the nodes when they are not needed.
+To configure Karpenter, you create *provisioners* that define how Karpenter manages unschedulable pods and expires nodes.
+Here are some things to know about the Karpenter provisioner:
+
+* **Unschedulable pods**: Karpenter only attempts to provision pods that have a status condition `Unschedulable=True`, which the kube scheduler sets when it fails to schedule the pod to existing capacity.
+
+* **Provisioner CR**: Karpenter defines a Custom Resource called a Provisioner to specify provisioning configuration.
+Each provisioner manages a distinct set of nodes, but pods can be scheduled to any provisioner that supports its scheduling constraints.
+A provisioner contains constraints that impact the nodes that can be provisioned and attributes of those nodes (such timers for removing nodes).
+See [Provisioner](/docs/provisioner-crd/) for a description of settings and the [Provisioning](/docs/tasks/provisioner.md) task for of provisioner examples. 
+
+* **Well-known labels**: The provisioner can use well-known Kubernetes labels to allow pods to request only certain instance types, architectures, operating systems, or other attributes when creating nodes.
+See [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) for details.
+Keep in mind that only a subset of these labels are supported in Karpenter, as described later.
+
+* **Deprovisioning nodes**: A provisioner can also include time-to-live values to indicate when nodes should be deprovisioned after a set amount of time from when they were created or after they becomes empty of deployed pods.
```

Comment:
```suggestion
* **Deprovisioning nodes**: A provisioner can also include time-to-live (TTL) values to indicate when nodes should be deprovisioned after a set amount of time from when they were created or after they becomes empty of deployed pods.
```

Does TTL affect both total time a node can exist and how long it should stay after empty? I thought TTL only affected total life of a node before it would be destroyed.

Reply:
I see my answer further below and realized you said "values" and not "value" which answers my question.